test/input.txt

```diff
@@ -23,74 +14,38 @@
   list
 
 ##########################################################
-# test add person command, setup state for further tests
+# test add person command, setup state for futuer tests
```

Comment:
Check if this change is relevant for this feature's PR, and remember to double check the spelling of words as well.